__tests__/app.test.js

```diff
@@ -78,4 +78,15 @@ describe('GET /api', () => {
             expect(result).toBe(true)
         })
     })
+})
+
+describe('GET /api/articles', () => {
+    test('GET: 200 sends and array of articles to the client sorted by date in descending order', () => {
+        return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then((response) => {
+            expect(response.body.articles).toBeSortedBy('created_at', { descending: true });
+        })
```

Comment:
Great start to testing, you need to add a few more please:
- A test to check that this responds with an array of article objects.
- What errors do you need to test for here?